charts/datadog/values.yaml

```diff
@@ -583,10 +592,10 @@ datadog:
   #   service.py: |-
 
   # datadog.dockerSocketPath -- Path to the docker socket
-  dockerSocketPath:  # /var/run/docker.sock
+  dockerSocketPath: # /var/run/docker.sock
```

Comment:
I think you need to keep 2 spaces before the comment to make the linter happy.

Reply:
fixed